### ISSUE TITLE
Fallback if image doesn't exist

### ISSUE THREAD

**Issue by rickard81** [tubalmartin/riloadr#5]
It would be nice to be able to set a fallback image if one of the breakpoint images doesn't exist. 

Example: 
I know that image1 always exists but I can't be sure that image2 exists. So instead of a 404 we show image1.

**Comment by tubalmartin** [comment]
Do you mean something like this?

``` js
var group = new Riloadr({
    breakpoints: [
        {name: 'exists', maxWidth: 320}, // All images of this size exist 
        {name: 'may_not_exist', minWidth: 321, maxWidth: 480, fallback: 'exists'} // Images of this size may not exist so if this is the case, use a fallback size.
        {name: 'may_also_not_exist', minWidth: 481, fallback: 'may_not_exist'} // Images of this size may not exist so if this is the case, use a fallback size. If the fallback size does not also exist, try the next fallback size if set (cascading system).
    ]
});
```

**Comment by rickard81** [comment]
Yeah, or maybe just the possiblilty to make one of the images default.

```
var group = new Riloadr({
    breakpoints: [
        {name: 'exists', maxWidth: 320, default: true},
        {name: 'may_not_exist', minWidth: 321, maxWidth: 480}
        {name: 'may_also_not_exist', minWidth: 481}
    ]
});
```

**Comment by tubalmartin** [comment]
Done! I've just released version 1.3.0. Check it out!

**Comment by rickard81** [comment]
That's awesome man! Thanks!
